src/scripts/genXplatEventing.py

```diff
@@ -282,18 +282,17 @@ def generateClrallEvents(eventNodes,allTemplates):
         #generate EventEnabled
         clrallEvents.append("inline BOOL EventEnabled")
         clrallEvents.append(eventName)
-        clrallEvents.append("() {return XplatEventLogger::IsEventLoggingEnabled() && EventXplatEnabled")
-        clrallEvents.append(eventName+"();}\n\n")
+        clrallEvents.append("() {return ")
+        clrallEvents.append("EventPipeEventEnabled" + eventName + "() || ")
+        clrallEvents.append("(XplatEventLogger::IsEventLoggingEnabled() && EventXplatEnabled")
```

Comment:
The LTTng generated code right now looks like this:

extern "C" BOOL  EventXplatEnabledGCStart(){ return tracepoint_enabled(DotNETRuntime, GCStart); }

Because this change refactors the EventEnabled calls, the LTTng EventXplatEnabled<Event> functions need to check XplatEventLogger::IsEventLoggingEnabled() first.  I think that this may need to happen in the clretwallmain.h generated functions because XplatEventLogger is in the VM and this code is in the PAL.